### ISSUE TITLE
[accessibility] Buttons (`<a href="#">` controls) should have `role="button"`

### ISSUE THREAD

**Issue by Malvoz** [MapBBCode/mapbbcode#64]
Looking at the demo page, I noticed the `<a href="#">` controls are missing [`role="button"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).

**Comment by Malvoz** [comment]
Other resources that you may find helpful:

- [Leaflet guide to accessibility: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
